reference/docs-conceptual/learn/deep-dives/everything-about-hashtable.md

```diff
@@ -884,7 +884,7 @@ used with member access (`.`) notation. Or, you can use array index (`[]`) notat
 
 ### $PSBoundParameters
 
-[$PSBoundParameters][$PSBoundParameters] is an automatic variable that only exists inside the context of a function.
+`$PSBoundParameters` is an automatic variable that only exists inside the context of a function.
```

Comment:
```suggestion
[$PSBoundParameters[PSBoundParameters] is an automatic variable that only exists inside the context of a function.
```

This is actually a link id that can be foun down at the bottom of this page. however the links didn't match the one below and so the built versions didn't look right. Could you commit this suggestion and that should look better.